### PR TITLE
Bump/sdl/all

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **sdl/***

---

- [x] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by ericLemanissier** [comment]
blocked by:
- https://github.com/conan-io/conan-center-index/pull/19984
- https://github.com/conan-io/conan-center-index/pull/19983

File: recipes/sdl/all/conanfile.py [hunk 1]
```diff
@@ -184,11 +184,11 @@ def build_requirements(self):
             # set. This could be because you are using a Mac OS X version less than 10.5
             # or because CMake's platform configuration is corrupt.
             # FIXME: Remove once CMake on macOS/M1 CI runners is upgraded.
-            self.tool_requires("cmake/3.25.3")
+            self.tool_requires("cmake/3.27.5")
```
Comment (StellaSmith) [inline recipes/sdl/all/conanfile.py:187]:
```suggestion
            self.tool_requires("cmake/[>3.5 <4]")
```
https://github.com/conan-io/conan-center-index/blob/master/docs/adding_packages/dependencies.md#version-ranges

**Review by StellaSmith — APPROVED** [review]

**Comment by ericLemanissier** [comment]
The build is now finished. You can take a look at the results [here](https://github.com/ericLemanissier/proof-of-conan/actions/runs/6415115696)

**Comment by ericLemanissier** [comment]
I don't understand this error, don't have time to work on it

**Pull request closed without merging**